### PR TITLE
Fix current DML&DDL corner case for bidirectional replication  (#890)

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
pick #890 

### What is changed and how it works?
pick #890 

### Check List 

Related changes

 - Need to update the documentation

### CODE REVIEW

File: pkg/loader/load.go [hunk 1]
```diff
@@ -300,6 +302,10 @@ func (s *loaderImpl) refreshTableInfo(schema string, table string) (info *tableI
 	return
 }
 
+func (s *loaderImpl) evitTableInfo(schema string, table string) {
```
Comment (suzaku) [inline pkg/loader/load.go]: In `master`, we already have a method called `invalidateTableInfo` for invalidating the cached table info.

Comment (july2993) [reply pkg/loader/load.go]: [remove invalidateTableInfo
](https://github.com/pingcap/tidb-binlog/pull/892/commits/7aa677b6cce31bf0d5439b94e219c52dda5e5ed7)

Comment (suzaku) [reply pkg/loader/load.go]: I suppose we should remove `evitTableInfo` instead, `evit` is not even an English word ...

Comment (july2993) [reply pkg/loader/load.go]: https://github.com/pingcap/tidb-binlog/pull/892/commits/686ef25a34a4c242fd8e29bd1ffb3ab89bcd93e4
use `evict`

**Comment by july2993** [comment]
@suzaku @GregoryIan PTAL

**Pull request merged**